packages/patternfly-4/react-core/src/components/Progress/ProgressContainer.js

```diff
@@ -0,0 +1,71 @@
+import React, { Fragment } from 'react';
+import styles from '@patternfly/patternfly-next/components/Progress/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import { CheckCircleIcon, TimesCircleIcon } from '@patternfly/react-icons';
+import ProgressBar from './ProgressBar';
+
+export const ProgressMeasureLocation = {
+  outside: 'outside',
+  inside: 'inside',
+  top: 'top',
+  none: 'none'
+};
+
+export const ProgressStatus = {
+  danger: 'danger',
+  success: 'success',
+  info: 'info'
+};
+
+const propTypes = {
+  /** Progress component DOM ID. */
+  id: PropTypes.string.isRequired,
+  /** Progress title. */
+  title: PropTypes.string,
+  /** Type of progress status. */
+  status: PropTypes.oneOf(Object.keys(ProgressStatus)),
+  /** Location of progress value. */
+  measureLocation: PropTypes.oneOf(Object.keys(ProgressMeasureLocation)),
+  /** Actual progress value. */
+  value: PropTypes.number.isRequired
+};
+
+const defaultProps = {
+  status: ProgressStatus.info,
+  measureLocation: ProgressMeasureLocation.Top,
+  title: ''
+};
+
+const statusToIcon = {
+  [ProgressStatus.danger]: TimesCircleIcon,
+  [ProgressStatus.success]: CheckCircleIcon
+};
+
+const ProgressContainer = ({ value, title, id, status, measureLocation }) => {
+  const StatusIcon = statusToIcon.hasOwnProperty(status) && statusToIcon[status];
+  return (
+    <Fragment>
```

Comment:
I think with Babel 7 you can use `<>` now. That will make the import not required.

Reply:
Jest complaines about unexpected token :disappointed: